From: =?utf-8?q?Zbigniew_J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sun, 1 Jun 2014 14:01:23 -0400
Subject: keyboard: add Plantronics .Audio mute button

https://bugs.freedesktop.org/show_bug.cgi?id=79495
(cherry picked from commit 9e3dbf6b2b99d0e16989d9cedb458729db5a60c3)
(cherry picked from commit db75f7e5b9f699ebd4936e9342482b6ffb67cb78)
---
 hwdb/60-keyboard.hwdb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hwdb/60-keyboard.hwdb b/hwdb/60-keyboard.hwdb
index b4079491..17f06e82 100644
--- a/hwdb/60-keyboard.hwdb
+++ b/hwdb/60-keyboard.hwdb
@@ -864,6 +864,14 @@ keyboard:dmi:bvn*:bvr*:bd*:svnOQO*Inc.*:pnOQO*Model*2*:pvr*
  KEYBOARD_KEY_f3=volumeup
 
 ###########################################################
+# Plantronics
+###########################################################
+
+# Plantronics .Audio 626 DSP
+keyboard:usb:v047fpC006*
+ KEYBOARD_KEY_b002f=f20                                # Microphone mute button; should be micmute
+
+###########################################################
 # Quanta
 ###########################################################
 
